README.md

```diff
@@ -45,14 +45,24 @@
   - [Input Output Model](#input-output-model)
   - [Circular Economy and Waste](#circular-economy-and-waste)
 - [Biosphere](#biosphere)
```

Comment:
Would it make sense to replace the `Biosphare` category completely. Instead making other categories with subcategories like for `Biodiversity` or `Forest`

- Biodiversity
  -  Biodiversity and Species Distribution
  - Biodiversity Data Access and Management
  - ...
- Forest Studies
  - Forest Remote Sensing
  - Forest Observation and Management
  - ...
 

and directly a new subcategory where no category is necessary like in `Ecological and Environmental Modeling`?

Reply:
Yes, that would be an option. But then we would have to do the same for the other spheres like hydrosphere or atmosphere.  From a strategic point of view, I would suggest that we start by further dividing the subcategories and after that check which subcategories we can make a new category.  We could also get rid of the consumption categories and make the subcategories new categories. The finer division of the categories would also improve ClimateTriage.com a lot.